### PR TITLE
Feat/add access to settings from LLM/LLD welcome screen

### PR DESCRIPTION
**Pull request by pierrelouis-c**
### 📝 Description

This PR adds a new entry point for the settings in LLD and LLM welcome screen.

### ❓ Context

- **Impacted projects**: `ledger-live-desktop` `ledger-live-mobile` 
- **Linked resource(s)**: [FAT-552] [FAT-553] 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

https://user-images.githubusercontent.com/91890529/198320950-755a9503-f945-4cf1-8023-77a66d488dfa.mp4

https://user-images.githubusercontent.com/91890529/198321381-dedfef19-4fb2-46ab-a9af-f7cddd1e699c.mov

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

[FAT-552]: https://ledgerhq.atlassian.net/browse/FAT-552?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FAT-553]: https://ledgerhq.atlassian.net/browse/FAT-553?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/screens/settings/WelcomeScreenSettings.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { RouterHistory, Match, Location } from "react-router-dom";
```
Comment (alexandremgo) [inline apps/ledger-live-desktop/src/renderer/screens/settings/WelcomeScreenSettings.tsx:2]: I don't know if it's my local packages, but it seems those 3 types does not exist or are not exported from `react-router-dom`

Comment (ofreyssinet-ledger) [reply apps/ledger-live-desktop/src/renderer/screens/settings/WelcomeScreenSettings.tsx:2]: :feelthepainharold:

Comment (alexandremgo) [reply apps/ledger-live-desktop/src/renderer/screens/settings/WelcomeScreenSettings.tsx]: thanks for the changes !

**Review by alexandremgo — APPROVED** [review]
QA will be happy 🤗

**Pull request merged**